foxy/distribution.yaml

```diff
@@ -3415,13 +3415,10 @@ repositories:
       url: https://gitlab.com/ApexAI/performance_test.git
       version: 1.0.0
```

Comment:
Now that this is out of sync hold I think it is blocked on the same comment as the other distributions. https://github.com/ros/rosdistro/pull/33763#discussion_r916281228